pkg/vault/test/authorizer/TimelockAuthorizerPermissions.test.ts

```diff
@@ -665,6 +598,48 @@ describe('TimelockAuthorizer permissions', () => {
             });
           });
         });
+
+        context('when the user has the permission in a specific contract and everywhere', () => {
+          sharedBeforeEach('grants the permissions', async () => {
+            await authorizer.grantPermission(ACTION_1, user, WHERE_1, { from: root });
+            await authorizer.grantPermissionGlobally(ACTION_1, user, { from: root });
```

Comment:
For completion, could check that it can do both after the grant, contrasted with tests below:
expect(await authorizer.canPerform(ACTION_1, user, WHERE_1)).to.be.true;
expect(await authorizer.canPerform(ACTION_1, user, WHERE_2)).to.be.true;
expect(await authorizer.canPerform(ACTION_1, user, EVERYWHERE)).to.be.true;